.github/actions/build-sdist/action.yml

```diff
@@ -0,0 +1,62 @@
+name: 'Build Source Distribution'
+description: 'Build a source distribution for an ofrak package'
+inputs:
+  package-name:
+    description: 'Name of the package to build'
+    required: true
+  source-dir:
+    description: 'Source directory containing the package'
+    required: true
+  lfs:
+    description: 'Whether this package requires LFS'
+    required: false
+    default: 'false'
+  python-version:
+    description: 'Python version to use for building'
+    required: true
```

Comment:
It seems like we only use Python 3.9 for this workflow (as far as I can tell). Should we just set a default here and make this input optional?